arc.tex

```diff
@@ -0,0 +1,110 @@
+Details on the \gls{USDF} specifications are given in \citeds{DMTN-189}.
+\action{GS,FH}{March 2022}{Should we have Specs like \gls{DMTN}-189 for UK and FRdF ?}
+
+\secref{sec:desc} gives a high level overview of the system, architecturally we look
+at this as a set of enclaves.
+As images are processed in the Prompt and Offline Production enclaves, their resulting data products are stored in the \gls{Archive} enclave and made available to the DAC enclave where data rights holders can access and analyze them.
+In addition, Rubin Observatory staff will use the Development/Integration enclave to maintain the Observatory's software tools and systems and to develop new versions of them.
+
+These enclaves are further described here and for each a series of subsections explore :
+
+
+\begin{enumerate}
+\item Threats and Security infrastructure
+\item Disaster recovery
+\end{enumerate}
+
+\subsection{Prompt \gls{Enclave}}
+
+The Prompt enclave receives images from the Observatory facilities in Chile via a Long Haul Network connection.
+It stores these and processes them into Prompt data products of three main types:
+alerts for things that have moved or changed, measurement catalogs, and processed images.
+The alerts can be further subdivided into \emph{streak} alerts for objects that have moved a long distance and \emph{non-streak} alerts for all other objects.
+Measurements in the catalogs follow the same subdivision. Images may be \gls{Commissioning} images used for testing and characterizing the Observatory systems, normal science images without significant \emph{streaks}, or delayed science images that do contain significant \emph{streaks}.
+
+\emph{Streak} alerts are transmitted to an \gls{Alert} Vetting System (AVS) located at a Trusted Broker Facility at  Lawrence Livermore National Lab.
+\emph{Non-streak} alerts and \emph{streak} alerts approved by the \gls{AVS} are to be published to the world at large within 60 seconds of the original raw image being taken.
+Normal science images are made available to data rights holders in the \gls{DAC} after an 80 hour embargo period.
+Delayed science images, as identified by the \gls{AVS}, and any unapproved \emph{streak} measurements taken from them are made available after a 10 day embargo period. Commissioning images are made available to data rights holders after a 30 day embargo period.
+
+All Prompt data products are checked for quality by automated systems but also by human operators from the Rubin Observatory staff, who have access to all images and data products in order to perform this function.
+
+\subsubsection{ Threats and Security infrastructure}
+The obvious threat surfaces here are :
+\begin{enumerate}
+\item Transmission of Data from Chile. IPSec built into the routers will be used on the \gls{LHN}. \citeds{DMTN-108} discusses threats in this realm a little more.
+\item Transmission to \gls{LLNL}.  This will be over internet using TLS.
```

Comment:
Is this going over some kind of DOE network like ESnet? Cause IPSecing the images to SLAC and then doing all the work to extract the information a bad guy would be most interested in and then using TLS to transmit it to Livermore implies to me that TLS is adequate for image transfer?

Reply:
It would be ESnet, but beyond that it could be that a physical network on US soil is harder to infiltrate.

Reply:
We did try to have TLS also from Chile but OGAs said not